ladybug/datacollection.py

```diff
@@ -95,6 +99,25 @@ def header(self):
     def header(self, h):
         self._header = None if not h else Header.from_header(h)
 
+    @property
+    def middle_hour(self):
+        """Get or set whether the values are interpreted as falling on the half hour.
+
+        This is a boolean that sets whether the values in the data collection
+        are interpreted as falling on the half hour (True) or the hour (False).
+        The default is set to False to be interpreted as fallin on the hour.
+        Note that all EnergyPlus and Radiance results are totalled or averaged
+        values over the hour and so are better characterized by having this
+        input set to True."""
+        return self._middle_hour
+
+    @middle_hour.setter
+    def middle_hour(self, mh):
```

Comment:
I originally thought it would be important but now that it's just a normal attribute on the header (like the other header attributes), it's definitely unnecessary.